src/app.js

```diff
@@ -64,6 +64,15 @@ passport.serializeUser((user, done) => {
   done(null, user.id);
 });
 
+passport.deserializeUser((id, done) => {
+  User.findByPk(id)
+    .then((user) => done(null, user))
+    .catch((err) => done(err, null));
+});
+
+// Routes
+app.use("/api", authRoute);
```

Comment:
This line <<app.use("/api", authRoute);<< is duplicated

Reply:
removed